conformance_tests/sysman/test_sysman_device/src/test_sysman_device.cpp

```diff
@@ -127,6 +127,55 @@ TEST_F(
   run_device_hierarchy_child_process();
 }
 
+TEST_F(
+    SYSMAN_DEVICE_TEST,
+    GivenValidDeviceWhenRetrievingSubDevicePropertiesThenValidPropertiesAreReturned) {
+  for (auto device : devices) {
+    auto device_properties = lzt::get_sysman_device_properties(device);
+    uint32_t sub_devices_count = device_properties.numSubdevices;
+
+    if (sub_devices_count > 0) {
+      uint32_t num_subdevices = 0;
+      lzt::get_sysman_subdevice_properties(device, &num_subdevices);
+      EXPECT_EQ(sub_devices_count, num_subdevices);
+      auto sub_device_properties =
+          lzt::get_sysman_subdevice_properties(device, &num_subdevices);
+      for (uint32_t sub_device_index = 0; sub_device_index < num_subdevices;
+           sub_device_index++) {
+        EXPECT_LT(sub_device_properties[sub_device_index].subdeviceId,
+                  num_subdevices);
+        EXPECT_GT(sub_device_properties[sub_device_index].uuid.id[0], 0);
+      }
+    }
+  }
+}
+
+TEST_F(
+    SYSMAN_DEVICE_TEST,
+    GivenValidDriverWhenRetrievingDeviceHandleFromUUIDThenReturnedDeviceHandleShouldMatchCurrentDeviceHandle) {
+  uint32_t driver_count = 0;
+  EXPECT_EQ(ZE_RESULT_SUCCESS, zesDriverGet(&driver_count, nullptr));
```

Comment:
can lzt::get_default_zes_driver() be used here instead of this?

Reply:
Actually i found get_default_driver(), which wasn't working so had to use this. But now have changed to get_default_zes_driver().
Thanks